docs/.vuepress/components/LatestVersion.vue

```diff
@@ -18,6 +18,7 @@ onMounted(async () => {
       latestVersion = shieldsResponse.value.substring(1);
     }
     replaceInDOM(document.body, /\$latest-version/g, latestVersion);
+    replaceInDOM(document.body, /{{latestVersion}}/g, latestVersion);
```

Comment:
And now all other ${latest-version} strings will not be replaced

Reply:
This is an addition, not a replacement.

Reply:
Oh nevermind, I'm blind ^^